### ISSUE TITLE
Accuracy issue

### ISSUE THREAD

**Issue by sachinbisht1** [Deci-AI/super-gradients#1025]
![send](https://github.com/Deci-AI/super-gradients/assets/88220453/c7b8c37e-2cf9-41cf-83da-a74d97eeceed)

Why the accuracy is so low 

I ran a data validator also 

![Uploading send2.jpg…]()

**Comment by sachinbisht1** [comment]
![send2](https://github.com/Deci-AI/super-gradients/assets/88220453/30e5eee7-b10b-4488-8c4d-d9e356e6767d)
data validator

**Comment by harpreetsahota204** [comment]
Hi @sachinbisht1,

I'm not sure what you mean about the accuracy being low. Can you share some more information here so that I can reproduce it? Ideally, a colab notebook that I can look at and see what you're doing. 

If you can't provide that, can you share more information about:
• Your task
• Your dataset
• What model you're using

**Comment by sachinbisht1** [comment]
Hi @harpreetsahota204 
I have a image of a document page i want to detect and classify its component in different segment like title text image and so on 
I am using Yolo NAS L
and Object Detection

**Comment by Louis-Dupont** [comment]
The hyperparameters used by default are the ones that were found to generally work relatively well on downstream tasks, but are by no mean the best for any dataset. You could investigate this.

But to be honest, this can come from many different places, and it's hard to give an answer without a deeper context.

What we cannot really help you without is the code snipper; could you please share it?
Did you try other model and get way better results ?
Do you have training graphs to share ?
